lib/rucio/db/sqla/util.py

```diff
@@ -201,21 +209,24 @@ def create_root_account(create_counters=True):
     iaa4 = models.IdentityAccountAssociation(identity=identity4.identity, identity_type=identity4.identity_type, account=account.account, is_default=True)
 
     # Account counters
-    if create_counters:
-        create_counters_for_new_account(account=account.account, session=s)
-
-    # Apply
-    for identity in [identity1, identity2, identity3, identity4]:
-        try:
-            s.add(identity)
-            s.commit()
-        except IntegrityError:
-            # Identities may already be in the DB when running multi-VO conversion
-            s.rollback()
-    s.add(account)
-    s.commit()
-    s.add_all([iaa1, iaa2, iaa3, iaa4])
-    s.commit()
+    # sqlAlchemy 2.0 format
```

Comment:
If the goal is to migrate everything to the 'sqlalchemy 2.0' format, this format becomes the new norm. So no need to put this comment at each code block

Reply:
I put it there to remind myself. removed.